### ISSUE TITLE
Sample TODO android app

### ISSUE THREAD

**Issue by shantharaju** [shantharaju/AndroidTodoApp#1]
/cc @codepathreview @codepath

**Comment by nesquena** [comment]
Looks good! This was intended in part to give you an introduction to the general rhythm of this course. The course is entirely project based with an app being assigned each week and then due the following week. Each project builds on the last to help each engineer to learn all practical Android development and best practices as quickly as possible. We also do a code review for each submitted project once the program starts.

Nice job with the project, hope you found it as a helpful introduction and see you later today!

**Comment by shantharaju** [comment]
Hi Nathan, 
Thanks for reviewing my code. I was bit anxious since I missed the 10:00pm deadline. 
Glad that you asked us to do this exercise. Learned quite good amount of stuff while setting up the project. Look forward to see you in the class.

**Comment by nesquena** [comment]
Yes, we will talk about that in class today. To continue in the course as a participant, we expect future submissions to be before the deadline. If we get late submissions, we will ask that person to withdraw at that point and become a remote observer (not attend in class, no longer receive code reviews or work on a group project)

**Comment by shantharaju** [comment]
I think that is fair ask. Thanks for making an exception this time.
